tmcprototype/CspMasterLeafNode/setup.cfg

```diff
@@ -0,0 +1,21 @@
+[metadata]
+description-file = README.rst
+
+[aliases]
+# Define `python setup.py test`
+test=pytest
+
+[coverage:run]
+branch = True
+source = CspMasterLeafNode
+
+[tool:pytest]
+testpaths = test
+addopts = --cov=CspMasterLeafNode --json-report --json-report-file=htmlcov/report.json --cov-report term --cov-report html --cov-report xml
```

Comment:
Line is very long. Check if it exceeds max length limit.

Reply:
Done. The commit id is: https://github.com/ska-telescope/tmc-prototype/commit/530792d1807f498f367b47e6a1dada0e4a1cd50d